required/tools/varioref.dtx

```diff
@@ -1200,10 +1200,10 @@
   }}
 %    \end{macrocode}
 %    There are no good variants for German (I think and still think but
-%    this is a matter of taste :-).
+%    this is a matter of taste :-)).
 % \changes{v1.0b}{1994/01/31}{Replace in incorrect `def by `let}
 %    The following definitions were recently suggested to me but since
-%    the original are in for a long time i don't want to change them
+%    the original are in for a long time I don't want to change them
```

Comment:
"original" or "originals" ?

So maybe this shout be:

the originals have been there for ...